editor/property_selector.cpp

```diff
@@ -338,46 +338,26 @@ void PropertySelector::_item_selected() {
 	}
 	String name = item->get_metadata(0);
 
-	String class_type;
-	if (type != Variant::NIL) {
-		class_type = Variant::get_type_name(type);
-
-	} else {
-		class_type = base_type;
-	}
-
 	DocData *dd = EditorHelp::get_doc_data();
 	String text;
 
+	typedef Map<String, DocData::ClassDoc>::Element class_list_element;
```

Comment:
This typedef is unnecessary, just use the explicit type in the code as done in countless other places in this codebase. That's what we prefer.

Reply:
updated. Let me know if there is anything else.